src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -135,7 +144,11 @@ const CountdownTimer = () => {
               <button
                 className={style.timerControlButton}
                 aria-label="skip"
-                onClick={() => setView('content-selection')}
+                onClick={() => {
+                  const timeElapsed = (((settings.workInterval * 60000) - getTime()) / 60000).toFixed(2)
+                  recordFocusInterval(timeElapsed)
+                  setView('mood-rating-1')
+                }}
```

Comment:
Glad we did this skip functionality. We gotta design these things like they may be used by wild cats, and I think that any wild cat would be stoked that it saves their timer data even if they accidentally (or deviously) paw the skip button

Reply:
^^^^^ Very very smart!! I love the inclusion of this!